memstore/backfill_manager.go

```diff
@@ -150,6 +150,10 @@ func (r *BackfillManager) StartBackfill() ([]*UpsertBatch, int64, uint32) {
 	r.Lock()
 	defer r.Unlock()
 
+	utils.GetLogger().With("action", "Backfill", "table", r.TableName, "shard", r.Shard,
+		"lastRedoFile", r.LastRedoFile, "lastOffset", r.LastBatchOffset, "newRedoFile", r.CurrentRedoFile,
+		"newOffset", r.CurrentBatchOffset). Infof("Start backfill", r.TableName)
```

Comment:
Infof("Start backfill %s", r.TableName)

Reply:
better use logger fields instead of any infof-like format call